tests/php/src/DevTools/CallbackReflectionTest.php

```diff
@@ -84,12 +84,12 @@ public function data_get_source() {
 				'includes/amp-helper-functions.php',
 				ReflectionFunction::class,
 			],
-			'core_includes_wp_scripts_method'     => [
-				'WP_Scripts::print_scripts',
-				'WP_Scripts::print_scripts',
+			'core_includes_wp_user_static_method' => [
+				'WP_User::get_data_by',
```

Comment:
@westonruter As discussed in chat, `WP_Scripts::print_scripts` was no longer viable for being tested as it is not a static method, and non-static methods are not callable on PHP 8 (i.e. `is_callable` returns false).

Reply:
To maintain the printing aspect, you could use `_WP_Editors::print_default_editor_scripts` instead here.

Reply:
Or maybe not. This is fine.

Reply:
`_WP_Editors::print_default_editor_scripts` was not callable for some reason, I didn't spend much time looking into it.

Reply:
I think it's because:

```
Private, not included by default. See wp_editor() in wp-includes/general-template.php.
```